core/templates/css/.stylelintrc

```diff
@@ -1,5 +1,8 @@
 { 
   "extends": "stylelint-config-recommended",
+  "plugins": [
```

Comment:
Add the same plugin in html stylelintrc as well.

Reply:
Done.